### PR TITLE
update authors and mailmap

### PR DESCRIPTION
**Pull request by jessfraz**
ping @tianon 

immediately regretted starting this, but then couldnt stop

i think i got em all

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -142,3 +142,30 @@ Jessica Frazelle <jess@docker.com> Jessie Frazelle <jfrazelle@users.noreply.gith
 Thomas LEVEIL <thomasleveil@gmail.com> Thomas LÉVEIL <thomasleveil@users.noreply.github.com>
 <oi@truffles.me.uk> <timruffles@googlemail.com>
 <Vincent.Bernat@exoscale.ch> <bernat@luffy.cx>
+Antonio Murdaca <antonio.murdaca@gmail.com> <me@runcom.ninja>
+Antonio Murdaca <antonio.murdaca@gmail.com> <runcom@linux.com>
+Antonio Murdaca <antonio.murdaca@gmail.com> <runcom@users.noreply.github.com>
+Darren Shepherd <darren.s.shepherd@gmail.com> <darren@rancher.com>
+Deshi Xiao <dxiao@redhat.com> <dsxiao@dataman-inc.com>
+Deshi Xiao <dxiao@redhat.com> <xiaods@gmail.com>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
```
Comment (duglin) [inline .mailmap]: what's the rule for when a "noreply" address is included?

**Comment by jfrazelle** [comment]
There is no rule the mail map is for finding duplicates not sending spam

**Comment by thaJeztah** [comment]
Thanks Jess, must've been a "fun" job :)

Looks good, but I'm not familiar with the way it's used to check for possible issues. Guess that means a "lgtm".

File: .mailmap (hunk 1 above)
Comment (estesp) [inline .mailmap]: it's listing any other addresses you have ever used so we can all agree there is only one Doug Davis no matter what address(es) you use.  So, I guess you used noreply.github.com somewhere as it shows up in a list of all mail addresses ever used.

**Comment by jfrazelle** [comment]
ping @tianon ;)

**Comment by tianon** [comment]
> immediately regretted starting this, but then couldnt stop

that feel, so much familiar

LGTM

**Pull request merged**